spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -66,3 +66,6 @@ definitions:
                   values:
                       - 0: System Healthy
                       - 1: An error has occurred
+        - protocol:
```

Comment:
The NAP FW string has already proved to be troublesome to use as a string programmatically - the first 100 lines of main.c in piksi_firmware are a NAP version comparison function (https://github.com/swift-nav/piksi_firmware/blob/master/src/main.c#L57).

I'd vote for making the SBP protocol version a tuple of `u8`s or something else thats easy to compare programmatically and is more efficient to send in frequent message such as the `MSG_HEARTBEAT`.

Standardization would be good eventually but I don't think we should get too deep into that right now? I'd say we should make the best choice for the SBP version and use that going forward and we can improve the other versioning schemes later.

For the FW versions I think strings make more sense anyway as they are primarily just for displaying to the user.